documentation/logs.md

```diff
@@ -74,7 +74,7 @@ Alarm ID | Severity | Description | Action
 [CHECK_MASTER](#check_master)|Major|A "slave" perseo has lost visibility of master.|Check the configured SMTP Server is accessible and working properly.
```

Comment:
This loop-protection functionality is nice at I think it should be described in the "main" documentation, not only in the piece about logs and alarms.

Maybe as an aditional short section after this one https://github.com/telefonicaid/perseo-fe/blob/master/documentation/architecture.md#action, if you don't think it is a better place.